sources/packages/backend/workflow/test/2021-2022/parttime-assessment/eligibility/parttime-assessment-2021-2022-eligibility-BCAG.e2e-spec.ts

```diff
@@ -103,4 +104,69 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-eligibility-BCAG
       });
     }
   });
+
+  it("Should determine provincialAwardNetBCAGAmount when awardEligibilityBCAG is true", async () => {
+    // Arrange
+    const assessmentConsolidatedData =
+      createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+    assessmentConsolidatedData.studentDataCRAReportedIncome = 20001;
+    assessmentConsolidatedData.studentDataRelationshipStatus = "married";
+    assessmentConsolidatedData.studentDataIsYourSpouseACanadianCitizen =
+      YesNoOptions.Yes;
+    assessmentConsolidatedData.partner1CRAReportedIncome = 22999;
+    // Public institution
+    assessmentConsolidatedData.institutionType = InstitutionTypes.BCPublic;
+    // Act
+    const calculatedAssessment = await executePartTimeAssessmentForProgramYear(
+      PROGRAM_YEAR,
+      assessmentConsolidatedData,
+    );
+    // Assert
+    // calculatedDataTotalFamilyIncome = 20001 + 22999
+    // awardEligibilityBCAG is true
+    // federalAwardBCAGAmount is greater than or equal to 100
+    expect(calculatedAssessment.variables.calculatedDataTotalFamilyIncome).toBe(
+      43000,
+    );
+    expect(calculatedAssessment.variables.calculatedDataFamilySize).toBe(2);
```

Comment:
I am not sure if the assertion of family size is required for the purpose of this test.